controllers/topics.controllers.js

```diff
@@ -1,8 +1,13 @@
 const {selectTopics} = require('../models/topics.models');
+const endpoints = require('../endpoints.json')
```

Comment:
We should not import this here. We should create a model that uses fs read file to read the content of the endpoints.json file 🙂